.husky/prepare-commit-msg

```diff
@@ -0,0 +1,16 @@
+#!/bin/sh
+set +e
+
+# Extract the current branch nam
+BRANCH_NAME=$(git symbolic-ref --short HEAD)
+
+# Define the regex pattern for the ticket ID
+TICKET_ID_REGEX="^[0-9]+"
```

Comment:
`^[0-9]{5,7}+` may be more certain it is a ticket id not just `2nd-attempt` false positives

Reply:
May delimit it with `-_` as well?

Reply:
Needs to `^[0-9]{5,8}`